etcdserver/server.go

```diff
@@ -1613,6 +1615,10 @@ func (s *EtcdServer) getCommittedIndex() uint64 {
 	return atomic.LoadUint64(&s.committedIndex)
 }
 
+func (s *EtcdServer) GetCommittedIndex() uint64 {
```

Comment:
exporting internal details (e.g., indexes) like this really should be avoided; it leads high coupling and poor cohesion